client/app/telecom/telephony/line/calls/filtering/telecom-telephony-line-calls-filtering.controller.js

```diff
@@ -1,4 +1,4 @@
-angular.module("managerApp").controller("TelecomTelephonyLineCallsFilteringCtrl", function ($stateParams, $q, $timeout, Toast, ToastError, OvhApiTelephony) {
+angular.module("managerApp").controller("TelecomTelephonyLineCallsFilteringCtrl", function ($stateParams, $q, $timeout, Toast, ToastError, OvhApiTelephony, $translate, telephonyBulk) {
```

Comment:
Clearly not a requirement but we used to prefer to stick with service injection starting with `$…`

```js
function ($stateParams, $q, $timeout, $translate, Toast, ToastError, OvhApiTelephony, telephonyBulk) {
```

Reply:
Woops, missed that one, thanks.